autozone-poc/src/components/Shops/Brand/Brand.js

```diff
@@ -0,0 +1,24 @@
+import React, { Fragment } from 'react'
+
+export default function Brand(props) {
+
+    function BrandItem(props) {
+        return (
+             <li><a href="#" >{props.value}</a></li>
```

Comment:
nesting of component should not be there.  create separate component for ITEM

Reply:
This page is not associated to me.